site2/docs/client-libraries-cgo.md

```diff
@@ -4,6 +4,8 @@ title: Pulsar CGo client
 sidebar_label: CGo(deprecated)
 ---
 
+> The CGo client has been deprecated since version 2.7.0. If possible, please use the [Go client](client-libraries-go.md) instead.
```

Comment:
```suggestion
> The CGo client has been deprecated since version 2.7.0. If possible, use the [Go client](client-libraries-go.md) instead.
```

Do not use please and thank you in technical information. Technical information requires an authoritative tone. Terms of politeness are superfluous, convey the wrong tone for technical material, and are not regarded the same way in all cultures. In marketing information, terms of politeness might be appropriate. Use the imperative mood in the first sentence of each step.
https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.x24xr2yx9b9t